doc/howtos/building-images.rst

```diff
@@ -122,6 +122,26 @@ with ``# require conf/distro/include/ostro-os-development.inc`` and
 uncomment it.
 
 
+Accelerating Build Time Using Shared-State Files Cache
+------------------------------------------------------
+
+As explained in the `Yocto Project Shared State Cache documentation`_, by design
+the build system builds everything from scratch unless it can determine that
+parts do not need to be rebuilt. The Yocto Project shared state code supports
+incremental builds and attempts to accelerate build time through the use
+of prebuilt data cache objects configured with the ``SSTATE_MIRRORS`` setting.
+
+By default, this ``SSTATE_MIRRORS`` configuration is disabled in :file:`conf/local.conf`
+but can be easily enabled by uncommenting the ``SSTATE_MIRRORS`` line
+in your :file:`conf/local.conf` file, as shown here:::
+
+   # Example for Ostro OS setup, recommended to use it:
+   #SSTATE_MIRRORS ?= "file://.* http://download.ostroproject.org/sstate/ostro-os/PATH"
```

Comment:
On Thu, 2016-03-10 at 04:50 -0800, Gustavo Sverzut Barbieri wrote:
> Likely not, but it is worth to keep a mention what is SSTATE, it's the
> core to deliver the precompiled binaries. 

But isn't SSTATE mirror as described here just a stop-gap measure? Just
configuring the mirrors suffers from poorer utilization of the SSTATE
because native tools still depend a lot on the host distribution.

eSDK will elevate that to some extend by running pre-compiled native
tools with a custom libc and thus avoid some of the host dependencies.